modules/ravestate_stalker/__init__.py

```diff
@@ -60,7 +60,7 @@ def create_subscriber(ctx: ContextWrapper):
             ctx.push(subscriber_parent.id(), face_names)
             ctx.push(subscriber_parent.id(), rec_faces)
 
-            @state(read=(face_names.id(), rec_faces.id()),  write=(rec_faces.id(), raw_out.id()))
+            @state(read=(face_names.id(), rec_faces.id()),  write=(rec_faces.id(), raw_out.id()), signal_name="lennart")
```

Comment:
```suggestion
            @state(read=(face_names.id(), rec_faces.id()),  write=(rec_faces.id(), raw_out.id()))
```
This was just an ugly hack for the finals